capstone-rs/src/ffi.rs

```diff
@@ -6,19 +6,18 @@ use libc::{self, c_char};
 /// Given a valid C-style, NUL terminated, UTF8-encoded string, returns a Rust `&str`
 ///
 /// Warnings:
-/// - No checks are made for: NULL pointers, valid UTF-8
+/// - No checks are made for: valid UTF-8
 /// - This function "creates" a reference with an arbitrary lifetime, so be careful to limit the
 ///   lifetime appropriately
+#[inline]
 pub(crate) unsafe fn str_from_cstr_ptr<'a>(ptr: *const c_char) -> Option<&'a str> {
-    if ptr.is_null() {
-        return None;
-    }
-
-    let len = libc::strlen(ptr);
+    (!ptr.is_null()).then(|| {
```

Comment:
I think the original if statement is more clear than `bool::then`

Reply:
Also, try to minimize the number changed lines to reduce the size of the change (where it makes sense)